app/include/dt-bindings/zmk/keys.h

```diff
@@ -5,142 +5,1385 @@
  */
 #pragma once
 
+#include <zmk/hid_usage.h>
```

Comment:
Actually just noticed this one *organiational* issue. In general, I've been keeping *all* headers that might be included into keymaps, and are by requirement *only* preprocessor directives, in the `include/dt-bindings/` directory so there is a clear delineation of "headers used in keymaps". Those headers of course can/will be used by the code, but it's clear they are used in keymaps this way.

@innovaker Any problems moving the couple files into those directories? Shouldn't require re-testing, beyond just verifying builds, and would help keep that distinction intact.

Reply:
To confirm:
- `zmk/hid_usage.h` to `dt-bindings/zmk/hid_usage.h`
- `zmk/hid_usage_pages.h` to `dt-bindings/zmk/hid_usage_pages.h`
?

Reply:
That's the only two I can see, yeah.

Reply:
Any preference on the `#include` orders for those?

Reply:
nevermind on my last

Reply:
Consider it done.